app/components/admin/menu_component.rb

```diff
@@ -3,28 +3,40 @@ class Admin::MenuComponent < ApplicationComponent
   use_helpers :can?
 
   def links
-    [
-      (proposals_link if feature?(:proposals)),
-      (debates_link if feature?(:debates)),
-      comments_link,
-      (polls_link if feature?(:polls)),
-      (legislation_link if feature?(:legislation)),
-      (budgets_link if feature?(:budgets)),
-      booths_links,
-      (signature_sheets_link if feature?(:signature_sheets)),
-      messages_links,
-      site_customization_links,
-      moderated_content_links,
-      profiles_links,
-      stats_link,
-      settings_links,
-      dashboard_links,
-      (machine_learning_link if ::MachineLearning.enabled?)
-    ]
+    if Rails.application.multitenancy_management_mode?
+      multitenancy_management_links
+    else
+      default_links
+    end
   end
 
   private
 
+    def default_links
+      [
+        (proposals_link if feature?(:proposals)),
+        (debates_link if feature?(:debates)),
+        (polls_link if feature?(:polls)),
+        comments_link,
```

Comment:
I think it must have been a mistake. I think it would only make sense to move it after the "budgets". But in order not to change things without being sure, we'd better keep it as it was.